source/_integrations/sms.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: "SMS"
```

Comment:
This title should match the title of the integration as set in the integration manifest file.
Right now, this is way to generic.

Reply:
You marked this conversation as resolved, but didn't change it.

Could you elaborate?

Reply:
I am sorry, don't understand what are you asking me todo.
Do you mean we need a new name?